### PR TITLE
Add react

### PR DESCRIPTION
**Pull request by coderbydesign**
- Adds a React front-end via compose
- Renames/reorganized files to be more sane

### CODE REVIEW

**Comment by SteveHNH** [comment]
Need to fix the import in seeds.py for it to startup in compose. 
`from app` => `from api`

**Review by SteveHNH — CHANGES_REQUESTED** [review]
Need to fix the import in seeds.py for it to startup in compose. 
`from app` => `from api`

**Comment by coderbydesign** [comment]
> Need to fix the import in seeds.py for it to startup in compose. `from app` => `from api`

@SteveHNH fixed in 4182fcf, nice catch!

File: api/Containerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi8/python-39
 
 WORKDIR /usr/src/app
```
Comment (SteveHNH) [inline api/Containerfile:3]: add `USER root` under here to support non-root containerized environments. Podman in particular runs in user space and npm is finicky about the permissions

**Pull request merged**